### ISSUE TITLE
Enable all php errors/notices by default in the PR sandboxes

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#4644]
Raising this issue after having discussed it during our weekly meeting earlier today.

We recently fixed #2200 which seems to be the cause of #4640 ...if error messages were enabled in our PR sandboxes, we could have picked these php notices, and would have fixed them as part of the PR that was merged. Same with #4641 I believe.

We should see if we are able to use config overrides in the settings.php file for PR sandboxes, and have all PHP errors shown.

**Comment by BWPanda** [comment]
Made a PR, but it doesn't work... Not sure why. https://github.com/backdrop/backdrop/pull/4088

**Comment by indigoxela** [comment]
> Not sure why.

Probably because you try to set a config variable, but that's not covered by settings_get().

**Comment by BWPanda** [comment]
> These settings allow you to specify values for anything stored in config within the files stored in the $config_directories variable above.

(https://github.com/backdrop/backdrop/blob/d60ef45b36be59dbbb216c6cfc1d52f7e8a71e25/settings.php#L431)

I thought *any* config value could be set in `settings.php` now...

**Comment by indigoxela** [comment]
Oops, you're of course right. That has been introduced in #2805.

I took a look at the sandbox and realized, that the telemetry module is **enabled**, which means that the .tugboat/settings.local.php file is completely ignored. When did that break?

**Comment by indigoxela** [comment]
Of course it's not completely ignored - the database settings are also in there and the trusted host pattern is set correctly.

**Comment by BWPanda** [comment]
Hmm, weird...

**Comment by BWPanda** [comment]
@indigoxela I looked into this, and it seems that Telemetry *is* disabled, it's just disabled from running on cron (the module's enabled, but the setting in settings.php prevents the cron run triggering it).

Likewise, the error config value in settings.php does work, it's just that Tugboat needs a rebuild when we change things like this.

So on that hunch I merged the change, rebuilt Tugboat's base preview, and sure enough it works now.
